lib/node_modules/@stdlib/blas/ext/base/dcusum/examples/c/example.c

```diff
@@ -27,17 +27,17 @@ int main( void ) {
 	double y[] = { 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 };
```

Comment:
@headlessNode Since you've removed `PRId64`, you also need to remove the `inttypes.h` header. And didn't catch this before, but since you've refactored to use `int`, you no longer need the `stdint.h` header. That should also apply to all your previous PRs.